### ISSUE TITLE
PHP-Serializer value view Error (0.8.0.3844)

### ISSUE THREAD

**Issue by amoogi** [redis/RedisDesktopManager#3506]
E.g.:

``` php
a:7:{i:0;O:8:"stdClass":4:
{s:10:"reward_srl";s:1:"1";s:15:"reward_prize_id";s:2:"74";s:11:"reward_disp";s:5:"50
개";s:9:"give_type";s:6:"marble";}i:1;O:8:"stdClass":4:
{s:10:"reward_srl";s:1:"2";s:15:"reward_prize_id";s:2:"75";s:11:"reward_disp";s:5:"50
개";s:9:"give_type";s:6:"marble";}i:2;O:8:"stdClass":4:
{s:10:"reward_srl";s:1:"3";s:15:"reward_prize_id";s:2:"76";s:11:"reward_disp";s:5:"50
개";s:9:"give_type";s:6:"marble";}i:3;O:8:"stdClass":4:
{s:10:"reward_srl";s:1:"4";s:15:"reward_prize_id";s:2:"77";s:11:"reward_disp";s:6:"100
개";s:9:"give_type";s:6:"marble";}i:4;O:8:"stdClass":4:
{s:10:"reward_srl";s:1:"5";s:15:"reward_prize_id";s:2:"78";s:11:"reward_disp";s:5:"80
개";s:9:"give_type";s:6:"marble";}i:5;O:8:"stdClass":4:
{s:10:"reward_srl";s:1:"6";s:15:"reward_prize_id";s:2:"79";s:11:"reward_disp";s:5:"80
개";s:9:"give_type";s:6:"marble";}i:6;O:8:"stdClass":4:
{s:10:"reward_srl";s:1:"7";s:15:"reward_prize_id";s:2:"80";s:11:"reward_disp";s:6:"300
개";s:9:"give_type";s:6:"marble";}}
```

=> "Error: Invalid PHP Serialized String: TypeError: Cannot read property 'SyntaxError' of undefined"

English only support?

**Comment by SergeC** [comment]
Just installed version 0.8.1-114 and I have the same issue with PHP serialized values. HEX view only works. I'm using Ubuntu 14.04 LTS

**Comment by uglide** [comment]
Fixed in 0.8.2
